.gitignore

```diff
@@ -30,19 +30,11 @@ classes
 *.swp
 # buffered files generated by editors ending in ~
 *~
-# testsuite generated files/folders
-PutObjectStoreDirHere
```

Comment:
My local env used to be full of PutObjectStoreDirHere and ObjectStore and index leftovers from query, like /query/person. Are they gone or why are we no longer gitignore them?

Reply:
They should be gone, but maybe not from all modules... if you notice new `PutObjectStoreDirHere` directories, please create an issue in JIRA